src/ui_lyrics_panel.h

```diff
@@ -5,6 +5,9 @@
 #include "img_processing.h"
 #include "lyric_io.h"
 #include "metadb_index_search_avoidance.h"
+#include <vector> 
+class LyricPanel;
+std::vector<LyricPanel*>& get_active_panels(); 

```

Comment:
I'm not super keen on returning the actual list of actual panels (and certainly not when neither of those things are `const`).

I wonder if there's a nice way to do this a little more directly...it feels like this should should be able to live in `ui_hooks` or somewhere like that...or that maybe there should be a central store for "the current lyrics"... I suppose until now most other use-cases have been powered by a particular panel (each of which already has their own lyrics on-hand).

I recognise this is possibly a difficult thing for a first-time contributor to reason about though because it'd be a  very much a zoomed-out-high-level-code-organisation sort of refactoring.

I can see if I get some time to take a look at trying that though, and I'm open to ideas.